pandas_gbq/gbq.py

```diff
@@ -421,6 +439,17 @@ def sizeof_fmt(num, suffix='B'):
             num /= 1024.0
         return fmt % (num, 'Y', suffix)
 
+    def query_price_for(self, bytes_num):
+        figlet = None
+        price = bytes_num * self.query_price_for_TB / 2**40
```

Comment:
I think it's reasonable to put the price estimate in, but note that it won't always apply since there is also a fixed price plan.

I'm not sure about the fancy text rendering. It's fun, but is it worth the extra dependency?